### ISSUE TITLE
Flaky test failure in `ReadOutputOfContainedBank.lf`

### ISSUE THREAD

**Issue by petervdonovan** [lf-lang/lingua-franca#1808]
This was observed in a branch that modifies the C generator. Also, when I reran the tests, they passed.

```
    +---------------------------------------------------------------------------+
    Failed: src\multiport\ReadOutputOfContainedBank.lf
    -----------------------------------------------------------------------------
    Reason: Test did not pass.
    Reported issues:
    Exit code: 101
    Compilation output:
    Generating code for: file:/D:/a/lingua-franca/lingua-franca/test/Rust/src/multiport/ReadOutputOfContainedBank.lf
    ******** mode: STANDALONE
    ******** generated sources: D:\a\lingua-franca\lingua-franca\test\Rust\src-gen\multiport\ReadOutputOfContainedBank
    --- Current working directory: D:\a\lingua-franca\lingua-franca\test\Rust\src-gen\multiport\ReadOutputOfContainedBank
    --- Executing command: cargo build --message-format json-diagnostic-rendered-ansi
    --- Current working directory: D:\a\lingua-franca\lingua-franca\test\Rust\src-gen\multiport\ReadOutputOfContainedBank
    --- Executing command: cargo metadata --format-version 1
    SUCCESS (compiling generated Rust code)
    Generated source code is in D:\a\lingua-franca\lingua-franca\test\Rust\src-gen\multiport\ReadOutputOfContainedBank
    Compiled binary is in D:\a\lingua-franca\lingua-franca\test\Rust\bin\multiport

    Execution output:
    thread '<unnamed>' panicked at 'called `Option::unwrap()` on a `None` value', src\reactors\read_output_of_contained_bank.rs:24:40
    note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

    +---------------------------------------------------------------------------+
```

**Comment by lhstrh** [comment]
Thanks for creating the issue -- I had already alerted @oowekyala about this.

**Comment by oowekyala** [comment]
Thanks, I think this was fixed by lf-lang/reactor-rs#43. We still need to update the reactor-rs head in this repo.
